src/domains/my/detail/components/PostList.jsx

```
@@ -3,7 +3,7 @@ import { useState } from 'react'
 import useBoardStore from '../../../../store/mypage/useBoardStore'
 import BoardCategory from '../../../../components/web/BoardCategory'
 
-const PostList = ({ mockData }) => {
+const PostList = ({ boardData }) => {
```

ESLint

[error] 'boardData' is missing in props validation

(react/prop-types)